src/components/PracticeSection/MultipleChoice.js

```diff
@@ -478,19 +478,19 @@ export default function MultipleChoice() {
   const getExplanationReward = (length) => {
     switch (length) {
       case 5: {
```

Comment:
all these verbose `case` statements... 
i know you didn't create them, and you inherited them...
but I just hope you know that they can be written in a more compact way, by defining them in a dictionary, 

    const EXPLANATION_REWARDS = {5: "ywFMR01L0", 20: "ywFMR01L0", ...}

then later you can simply write 

```
  function getExplanationReward(length) {
    if length in  EXPLANATION_REWARD then {
      return EXPLANATION_REWARD[length]}
   else {return " " } 
```

or something like that .

Reply:
I will be aware of this